1001-2000/LIT1960Mashaf.xml

```diff
@@ -114,11 +115,55 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                   
                      <label>Evening Prayer of the Incense, from Egypt</label>
                   </div>
+               </div>
                
                
-               </div>
                
                
+               <div type="textpart" xml:id="PrayerIncense" corresp="LIT2252salota">
+                  <label>Prayers of the Incense (collection of prayers)</label>
+                  <!--This section follows the attestation in UppEt20. It is different from <ref corresp="#Incense"/>-->
```

Comment:
This information should be given in a note, which is visible to the user.

Reply:
In other parts of the record, it seems that this kind of information is given in "ausgeklammerten" comments

Reply:
(To be honest, the entire LIT1960Mashaf is quite messy. I can follow your suggestion for the part that I added: it will be unsystematic, but maybe it's rather the other parts that should be changed 👍 )

Reply:
The commenting out depends on whether this comment is just for you as the encoder, as a temporary reminder to encode something later, because you do not know whether and how to encode - then yes comment out, it is then not visible for anyone, not searchable, and will not be also present in the expanded TEI XML on the server.

if you want to put it as if it were a footnote, or a note in brackets, then it must be properly encoded as a `note `or similar element.

note that references to other elements in the same record should have the target `<ref target="#Incense">Prayers of the Incense</ref>`

This information can also be provided as a relation in LIT2252salota (`<relation name="saws:isDifferentTo" active="LIT2252salota" passive="LIT1960Mashaf#Incense">`)

Reply:
Thanks! My intention was to note this down for the future person who will clean up LIT1960Mashaf (based on a better understanding on this work than mine), but I turned it into a general note instead. The information might be relevant in general, as noted by Carsten.

Reply:
I added the relation in LIT2252salota according to your suggestion. Thank you! :)